models/twilio_message_enhanced.sql

```diff
@@ -1,4 +1,4 @@
-with messages as (
+with messages as ( 
```

Comment:
If we are keeping the `using_twilio_message` variable, then we will need to include a config block on this model to disable it if they are not using messages. 

What is the percent of customers that use messages? I worry that allowing customers to disable the messages component will result in a package with little to no value.

Reply:
That's true, it looks like the majority use it, so I actually think removing the config makes the most sense

Reply:
removed from source

Reply:
Actually, I realize that the data in our internal shared environment doesn't have the `messaging_service` object. Perhaps _should_ leave in the configs?

Reply:
I think we keep the removal of the config for messages, but add one specific for messaging_service as that is just a complementary addition.

Reply:
Great, added back the enabled config for messaging_service

Reply:
Thanks! If you add the config however you will also need to add conditionals for wherever you reference the messaging service.

For example this join should only be run if the variable is enabled.

https://github.com/fivetran/dbt_twilio/blob/54b5a438b8df78c95d3aedcdf57315383e00c754/models/twilio__message_enhanced.sql#L61-L62

Reply:
I saw you addressed this!